src/commands/init.ts

```diff
@@ -32,6 +31,7 @@ export default async () => {
 		}
 	};
 
-	const file = response.configFormat === 'json' ? JSON.stringify(config, null, 2) : YAML.stringify(config);
-	return writeFile(packageJson.replace('package.json', `.sapphirerc.${response.configFormat}`), file);
+	const file = response.configFormat === 'json' ? JSON.stringify(config, null, 2) : dump(config);
+	await writeFile(packageJson.replace('package.json', `.sapphirerc.${response.configFormat}`), file);
+	return process.exit(0);
```

Comment:
Isn't this process.exit redundant?

Reply:
It kiinda is but there are 2 reasons I added
1. I noticed a quicker exit out after it's done with this in there
2. It is required for `generate` to satisfy TS 7030 Not All code paths return a value. For consistency, I added it to all.